components/src/core/components/Topbar/Navigation.vue

```diff
@@ -109,7 +113,8 @@ export default defineComponent({
 
     const arrangeMenuItems = (menuItems: ComponentPublicInstance[]) => {
       // offset the width of "more" element
-      const widthOffset = hiddenMenuItems.value.length > 0 ? 80 : 0;
+      let widthOffset = hiddenMenuItems.value.length > 0 ? 80 : 0;
+      if (context.slots.default?.()) widthOffset += 35;
```

Comment:
```suggestion
      if (context.slots.default?.() != undefined) widthOffset += 35;
```
do we need something like this

Reply:
since `undefined` get assessed to false this should be fine